var/spack/repos/builtin/packages/rocrand/package.py

```diff
@@ -21,7 +21,7 @@ class Rocrand(CMakePackage):
 
     maintainers = ["cgmb", "srekolam", "renjithravindrankannath"]
     libraries = ["librocrand"]
-
+    version("master", branch="develop")
```

Comment:
I would suggest

    version("develop", branch="develop")
    version("master", branch="master")

Reply:
Thank you @cgmb.   Updated as per the suggestion.